src/lib/connectors/repos/events.js

```diff
@@ -1,12 +1,12 @@
-const { Event } = require('../bookshelf');
-
+const { Event, bookshelf } = require('../bookshelf');
+const queries = require('./queries/events');
 /**
  * creates a new event record in the database
  * @param {Object} event event pojo
  * @returns {Object} event pojo
  */
 const create = async (event) => {
-  const result = await Event.forge(event).save();
+  const result = await Event.forge().save(event);
```

Comment:
Interestingly both methods works. I followed the billing batch transactions approach to remain consistent. Did it not work the other way for you?

Reply:
That's odd. The first way didn't work for the returns notifications feature, it only worked after the change I made

Reply:
I had a second look at this after I figured out what was going wrong with this event and it does work the way you had it, so I've changed it back